lib/src/components/NxToast/NxToastContainer.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useRef} from 'react';
+import FocusContext from './contexts';
+
+import { NxToastContainerProps, nxToastContainerPropTypes } from './types';
+
+export { Props } from './types';
+
+const NxToastContainer = (props: NxToastContainerProps) => {
+  const { children } = props;
+
+  const ref = useRef<HTMLDivElement | null>(null),
+      prevSelectedRef = useRef<Element | null>(null);
+
+  const body = document.getElementsByClassName('nx-body')[0];
+
+  const adjustFocus = () => {
+    //When closing Toasts with a mouse click, document.activeElement becomes nx-body, so need
+    //to specify not to assign selectedRef to body so the focus returns to the appropriate element
+    if (!ref.current?.contains(document.activeElement) && document.activeElement !== body) {
+      prevSelectedRef.current = document.activeElement;
+    }
```

Comment:
I don't think I understand this part exactly.  It seems like it's responsible for keeping track of the last most recently focused element outside of the toasts, but that can change in a wide variety of ways that don't fire this function.

Reply:
> I don't think I understand this part exactly. It seems like it's responsible for keeping track of the last most recently focused element outside of the toasts, but that can change in a wide variety of ways that don't fire this function.

focusing code is adjusted with a focusin event listener here: [a10a2d5](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/a10a2d51bd38d9fe3e6b555910ac9271eb3f99cb)